### ISSUE TITLE
With an off-white page background, text focus styles are hard to see

### ISSUE THREAD

**Issue by kjellr** [WordPress/twentytwentyone#873]
**Describe the bug**

If the page background is set to an off-white color (I used `#f9f9f9` in my testing), the focus styles for buttons and links is hard to see: 

**Expected behavior**

I expected something more akin to the dark focus background used when the background is set to white. 

**Screenshots**

Navigation:
![Screen Shot 2020-11-23 at 3 11 48 PM](https://user-images.githubusercontent.com/1202812/100010519-60613c80-2d9e-11eb-8ea2-58cb53337217.png)

Post title:
![Screen Shot 2020-11-23 at 3 14 48 PM](https://user-images.githubusercontent.com/1202812/100010715-ad451300-2d9e-11eb-9b89-a9eb2786c319.png)

Post meta:
![Screen Shot 2020-11-23 at 3 14 55 PM](https://user-images.githubusercontent.com/1202812/100010726-b0400380-2d9e-11eb-858e-677a5d60399b.png)

**Comment by carolinan** [comment]
We can adjust this function if we can decide at what luminance we need to change the link focus colors to black background and white text. 

None of the light palette colors live up to the 3:1 or 4.5:1 contrast requirements when we measure the contrast between the color and the white link background.

```
	public function body_class( $classes ) {
		$background_color = get_theme_mod( 'background_color', 'D1E4DD' );
		if ( 127 > self::get_relative_luminance_from_hex( $background_color ) ) {
			$classes[] = 'is-dark-theme';
		} else {
			$classes[] = 'is-light-theme';
		}

		if ( 'ffffff' === strtolower( $background_color ) ) {
			$classes[] = 'has-background-white';
		}

		return $classes;
	}
```
-I don't believe the `is-light-theme` class is used in the Customizer. It is a utility class and it is currently not used anywhere in the theme by default.

**Comment by carolinan** [comment]
The contrast ratio between the background colors for the default and the focused state needs to be 3:1.

4.5:1 for the background and text color.

**Comment by carolinan** [comment]
Yellow:
Foreground:#FFFFFF
Background:#EEEADD

The contrast ratio is: 1,2:1

Orange:
Foreground:#FFFFFF
Background:#E4DAD1

The contrast ratio is: 1,4:1

Red:
Foreground:#FFFFFF
Background:#E4D1D1

The contrast ratio is: 1,5:1

Purple:
Foreground:#FFFFFF
Background:#D1D1E4

The contrast ratio is: 1,5:1

Blue:
Foreground:#FFFFFF
Background:#D1DFE4

The contrast ratio is: 1,4:1

Green:
Foreground:#FFFFFF
Background:#D1E4DD

The contrast ratio is: 1,3:1


For these palette colors, a white background on focus is not enough, there would also need to be a text decoration change.
For links that have underlines, we can remove that underline on focus.
